### PR TITLE
Add BioPass FIDO2 to udev rules

### PR DESCRIPTION
**Pull request by fgervais**

### CODE REVIEW

File: u2f.conf.sample [hunk 1]
```diff
@@ -40,13 +40,13 @@ notify 100 {
...
 
-# Feitian ePass FIDO
+# Feitian ePass FIDO, BioPass FIDO2
 notify 100 {
 	match "system"		"USB";
 	match "subsystem"	"DEVICE";
 	match "type"		"ATTACH";
 	match "vendor"		"0x096e";
-	match "product"		"(0x0850|0x0852|0x0853|0x0854|0x0856|0x0858|0x085a|0x085b)";
+	match "product"		"(0x0850|0x0852|0x0853|0x0854|0x0856|0x0858|0x085a|0x085b|085d)";
```
Comment (klali) [inline u2f.conf.sample]: 085d needs to be prefixed with 0x here

Comment (fgervais) [reply u2f.conf.sample]: Good I made the change.

**Comment by fgervais** [comment]
I'm not too used to the review process.

Did I do things correctly?

**Comment by klali** [comment]
Yes, perfect.

Merging.

**Pull request merged**